### ISSUE TITLE
[Bug]: Manual video import fails due to old metadata info json from an old youtube-dl version

### ISSUE THREAD

**Issue by dot-mike** [tubearchivist/tubearchivist#581]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

Docker

## Describe the bug

Importing video manually fails. I have the video and the metadata-file from way back time but TA won't import it.
It was downloaded with an old version (2020~) of youtube-dl

### Relevant log output

```shell
2023/11/02 00:44:08	stderr	KeyError: 'tags'
2023/11/02 00:44:08	stderr	            ~~~~~~~~~~~~~~~~~^^^^^^^^
2023/11/02 00:44:08	stderr	    "tags": self.youtube_meta["tags"],
2023/11/02 00:44:08	stderr	  File "/app/home/src/index/video.py", line 194, in _process_youtube_meta
2023/11/02 00:44:08	stderr	    self._process_youtube_meta()
2023/11/02 00:44:08	stderr	  File "/app/home/src/index/video.py", line 149, in build_json
2023/11/02 00:44:08	stderr	    video.build_json(
2023/11/02 00:44:08	stderr	  File "/app/home/src/index/manual.py", line 402, in index_metadata
2023/11/02 00:44:08	stderr	                ^^^^^^^^^^^^^^^^^^^^^
2023/11/02 00:44:08	stderr	    json_data = self.index_metadata()
2023/11/02 00:44:08	stderr	  File "/app/home/src/index/manual.py", line 394, in run
2023/11/02 00:44:08	stderr	    ManualImport(current_video, self.CONFIG).run()
2023/11/02 00:44:08	stderr	  File "/app/home/src/index/manual.py", line 146, in process_videos
2023/11/02 00:44:08	stderr	    self.process_videos()
2023/11/02 00:44:08	stderr	  File "/app/home/src/index/manual.py", line 55, in scan
2023/11/02 00:44:08	stderr	    ImportFolderScanner(task=self).scan()
2023/11/02 00:44:08	stderr	  File "/app/home/tasks.py", line 271, in run_manual_import
2023/11/02 00:44:08	stderr	           ^^^^^^^^^^^^^^^^^^^^^^^^^
2023/11/02 00:44:08	stderr	    return self.run(*args, **kwargs)
2023/11/02 00:44:08	stderr	  File "/root/.local/lib/python3.11/site-packages/celery/app/trace.py", line 760, in __protected_call__
2023/11/02 00:44:08	stderr	                 ^^^^^^^^^^^^^^^^^^^^
2023/11/02 00:44:08	stderr	    R = retval = fun(*args, **kwargs)
2023/11/02 00:44:08	stderr	  File "/root/.local/lib/python3.11/site-packages/celery/app/trace.py", line 477, in trace_task
2023/11/02 00:44:08	stderr	Traceback (most recent call last):
2023/11/02 00:44:08	stderr	[2023-11-02 00:44:08,009: ERROR/ForkPoolWorker-4] Task manual_import[42f7bcc5-20f8-4e16-a0ea-cf062b9b503d] raised unexpected: KeyError('tags')
2023/11/02 00:44:08	stderr	[2023-11-02 00:44:08,006: WARNING/ForkPoolWorker-4] 42f7bcc5-20f8-4e16-a0ea-cf062b9b503d Failed callback
2023/11/02 00:44:07	stderr	[2023-11-02 00:44:07,996: WARNING/ForkPoolWorker-4] https://www.youtube.com/watch?v=B_ERb60XJs4: failed to get info from youtube with message ERROR: [youtube] B_ERb60XJs4: Private video. Sign in if you've been granted access to this video
2023/11/02 00:44:07	stderr	[2023-11-02 00:44:07,996: WARNING/ForkPoolWorker-4] ERROR: [youtube] B_ERb60XJs4: Private video. Sign in if you've been granted access to this video
2023/11/02 00:44:06	stderr	[2023-11-02 00:44:06,941: WARNING/ForkPoolWorker-4] B_ERb60XJs4: get metadata from youtube
2023/11/02 00:44:06	stderr	[2023-11-02 00:44:06,930: WARNING/ForkPoolWorker-4] manual import: {'media': '/cache/import/[B_ERb60XJs4].mp4', 'video_id': 'B_ERb60XJs4', 'metadata': '/cache/import/[B_ERb60XJs4].info.json', 'thumb': '/cache/import/[B_ERb60XJs4].jpg', 'subtitle': []}

```
```


### Anything else?

_No response_

**Comment by bbilly1** [comment]
info json format has changed over time. it's telling you what's missing: there is a `tags` expected. just add it to the json file, and you should be good.

**Comment by PhuriousGeorge** [comment]
Minimum viable json tags for importing are:
`{"id": "", "channel_id": "", "title": "",  "upload_date": "", "description": null, "categories": null, "thumbnail": null, "tags": null, "view_count": null}`

**Comment by bbilly1** [comment]
OK, if you insist, we can do some fallback values for these values. Also taking a PR.

**Comment by bbilly1** [comment]
released with v0.4.4
